### PR TITLE
Feature #37677 documentation : Exclude some groups from sharing with users

### PR DESCRIPTION
**Pull request by smarinier**
### ☑️ Resolves

* PR #38173 documentation

### 🖼️ Screenshots

image
image

### CODE REVIEW

**Review by artonge — COMMENTED** [review]
Maybe better?

File: admin_manual/configuration_files/file_sharing_configuration.rst [hunk 1]
```diff
@@ -18,7 +18,10 @@ Configure your sharing policy on your Admin page in the Sharing section.
   * Check ``Allow resharing`` to enable users to re-share files shared with them.
   * Check ``Allow sharing with groups`` to enable users to share with groups.
   * Check ``Restrict users to only share with users in their groups`` to confine
-    sharing within group memberships.
+    sharing within group memberships. When you check this, you'll
+    get an optional dropdown list of ignored groups when checking group membership.
+    Type any group name to search for.
+    Ignored groups are not used to determine if users may share with each others.
```
[the remark this answers is not in the document]
Comment (smarinier) [reply admin_manual/configuration_files/file_sharing_configuration.rst:24]: Arhhh... There is still a misunderstanding about this feature.
What you describe here (to my opinion) seems to be "Exclude groups from sharing" (further down in settings).
The use case is, when you select "Restrict users to only share with users in their groups", you may have some globals groups (eg "Admins" or "Guests"). But you don't want these "guests" or "admins" share each others.
User "John" is 'admin' and 'group1', he shouldn't be able to share with "Bob" 'admin' and 'group2', but with "Ted" wich is 'admin' and 'group1'. So the new settings allow you to set "admin" as a group that won't be used to determine who can share with who. (But 'group1' or 'group2' will).
There were a lot of debates to find a sentence to describe this. But we didn't find any short one !

[the remark this answers is not in the document]
Comment (artonge) [reply admin_manual/configuration_files/file_sharing_configuration.rst:24]: I dont' think so :).
My comment was mostly addressing the format of the text, to stay in line with the existing documentation.
Regarding the content, both your version or mine is fine with me, but I would favor mine.

[the remark this answers is not in the document]
Comment (smarinier) [reply admin_manual/configuration_files/file_sharing_configuration.rst:24]: I agree with format of the text. But the comment may confuse the admin, as users from added groups to this settings (eg in my example 'admin') may share or not : it won't depend on their membership to this group.

A sentence could be:
* Groups added to  ``Ignore the following groups when checking group membership`` won't be taken in account to determine if users are in same groups and may share with each others.

**Review by artonge — APPROVED** [review]

**Comment by artonge** [comment]
@smarinier can you fix the DCO?

**Comment by smarinier** [comment]
> @smarinier can you fix the DCO?

Rahhhh... ;)

**Pull request merged**